rq-worker/src/livery.py

```diff
@@ -149,14 +150,22 @@ def get_callsign_image(aviator_props, prop):
     if "font_alignment" in prop:
         draw.text_alignment = prop["font_alignment"]
 
+    if "font_color" in prop:
+        draw.fill_color = Color(prop["font_color"])
+
+    quotes = ""
+    if "callsign_quotes" in prop and prop["callsign_quotes"] == "1":
+        quotes = "\""
```

Comment:
you could simplify this statement to 

quotes = props.get("callsign_quotes", "")

the second parameter is what will be returned of the key doesn't exist in the dictionary.